### PR TITLE
Add --classes to evaluation of classification models

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

**Comment by sovrasov** [comment]
Looks like the CI is down:
```
ERROR: Error fetching remote repo 'origin'
```

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

File: pytorch_toolkit/ote/ote/modules/arg_parsers/reid.py [hunk 1]
```diff
@@ -32,6 +32,14 @@ def __init__(self):
...
     def get_train_parser(self, config_path):
         parser = argparse.ArgumentParser(parents=[train_args_parser(config_path)], add_help=False)
+        return self._add_custom_params(parser)
+
+    def get_test_parser(self, config_path):
+        parser = argparse.ArgumentParser(parents=[test_args_parser(config_path)], add_help=False)
+        return self._add_custom_params(parser)
+
+
+    def _add_custom_params(self, parser):
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/ote/ote/modules/arg_parsers/reid.py:42]: export and compression parsers?

Comment (sovrasov) [reply pytorch_toolkit/ote/ote/modules/arg_parsers/reid.py:42]: Forgot about them, gona add

**Comment by Ilya-Krylov** [comment]
@sovrasov your tests failed

**Comment by sovrasov** [comment]
I know. I have to run eval with an unitialized snapshot, so the results depend on random seed. Kirill changed random initialization and we have a different metric now.

**Comment by sovrasov** [comment]
Tests are passed

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**